pkg/shp/bundle/bundle.go

```diff
@@ -20,21 +20,22 @@ import (
 // GetSourceBundleImage returns the source bundle image of the build that is
 // associated with the provided buildrun, an empty string if source bundle is
 // not used, or an error in case the build cannot be obtained
-func GetSourceBundleImage(ctx context.Context, client buildclientset.Interface, buildRun *buildv1alpha1.BuildRun) (string, error) {
+func GetSourceBundleImage(ctx context.Context, client buildclientset.Interface, buildRun *buildv1beta1.BuildRun) (string, error) {
 	if buildRun == nil {
 		return "", fmt.Errorf("no buildrun provided, given reference is nil")
 	}
 
-	if buildRun.Spec.BuildRef != nil {
-		name, namespace := buildRun.Spec.BuildRef.Name, buildRun.Namespace
+	if buildRun.Spec.Build.Name != nil && *buildRun.Spec.Build.Name != "" {
```

Comment:
nit - ditto on use of len:

```suggestion
	if len(buildRun.Spec.Build.Name) > 0 {
```